src/main/groovy/nu/studer/gradle/plugindev/PluginDevPlugin.groovy

```diff
@@ -288,7 +292,9 @@ class PluginDevPlugin implements Plugin<Project> {
             publicDownloadNumbers = true
             version {
                 vcsTag = publication.version
-                attributes = ['gradle-plugin': "$extension.pluginId:$publication.groupId:$publication.artifactId"]
+                attributes = ['gradle-plugin': extension.pluginImplementations.collect {
```

Comment:
This is actually wrong

It looks like the gradle-plugin value needs to be an array instead of a string